defaults/main.yaml

```diff
@@ -303,3 +303,5 @@ grafana_config:
   log.console: "{{ grafana_log__console }}"
   log.file: "{{ grafana_log__file }}"
   event_publisher: "{{ grafana_event_publisher }}"
+
+grafana_apt_repo: deb https://packagecloud.io/grafana/stable/debian/ wheezy main
```

Comment:
Nevermind on my suggestion for `ansible_distribution_release`. It looks like Grafana doesn't actually generate the package per release and simply suggests to use `jessie` regardless of actual release.